engine/apps/alerts/models/alert_group.py

```diff
@@ -545,7 +546,7 @@ def web_link(self) -> str:
     @property
     def declare_incident_link(self) -> str:
         """Generate a link for AlertGroup to declare Grafana Incident by click"""
-        incident_link = urljoin(self.channel.organization.grafana_url, "a/grafana-incident-app/incidents/declare/")
+        incident_link = urljoin(self.channel.organization.grafana_url, f"a/{PluginID.INCIDENT}/incidents/declare/")
```

Comment:
I'll finish renaming the hardcoded plugin IDs, to use `PluginID`, in a subsequent PR